packages/core/src/transforms/resize.ts

```diff
@@ -41,7 +41,7 @@ function parseAspect(aspect: string): number | undefined {
   return aspectRatio
 }
 
-export const resize: TransformFactory<ResizeOptions> = (config) => {
+export const resize: TransformFactory<ResizeOptions> = (config, context) => {
```

Comment:
what is context and why is `resize` the only transform being changed?

Reply:
`resize` is the only transform that currently issues warnings.

`context` was some existing boilerplate already included for this purpose it seems. Implementing this functionality definitively gave me the impression that someone had already planned it to be implemented exactly like this. The `context` is created [here](https://github.com/JonasKruckenberg/imagetools/blob/main/packages/core/src/lib/generate-transforms.ts#L8).